category_exist_or_news_actived.md

```diff
@@ -0,0 +1,7 @@
+###10. Lấy Category có tồn tại blog hoặc news đã active (không được lặp lại category)
+```mysql
+SELECT distinct(category.id),category.title,category.description
+FROM category,blog,news
+WHERE category.id=blog.category_id 
```

Comment:
chỗ này mình dùng subquery where id in (Select category_id from blog where is_active = 1) OR id IN (....news)